highfive/configs/rust-lang/std-dev-guide.json

```diff
@@ -0,0 +1,14 @@
+{
+    "groups": {
+        "all": [
+            "@dtolnay",
+            "@joshtriplett",
+            "@Mark-Simulacrum",
+            "@kennytm",
+            "@yaahc",
+            "@m-ou-se"
+        ]
+    },
+    "contributing": "https://github.com/rust-lang/cargo/blob/master/CONTRIBUTING.md",
```

Comment:
This "contributing" line is for cargo.  I don't think it is really needed.

```suggestion
```

Reply:
lol yea, ty, I thought I updated that but I guess I got interupted...